lib/rex/post/meterpreter/extensions/stdapi/audio_output/audio_output.rb

```diff
@@ -0,0 +1,55 @@
+# -*- coding: binary -*-
+
+require 'rex/post/meterpreter/channel'
+require 'rex/post/meterpreter/channels/pools/stream_pool'
+
+module Rex
+  module Post
+    module Meterpreter
+      module Extensions
+        module Stdapi
+          module AudioOutput
```

Comment:
Minor: remove indention to be more consistent with other stdapi classes. For example, see `lib/rex/post/meterpreter/extensions/stdapi/fs/dir.rb`.